test/Function/StackArgsWithFormals.js

```diff
@@ -0,0 +1,141 @@
+//Test1 : Tests Optimization with bailout.
+var shouldBailout = false;

```

Comment:
convert this test to use the unittestframework. You can see other test in the es6 folder to get the idea.